test_runner/src/main/kotlin/ftl/cli/firebase/test/ios/IosCancelCommand.kt

```diff
@@ -0,0 +1,37 @@
+package ftl.cli.firebase.test.ios
+
+import ftl.args.IosArgs
+import ftl.config.FtlConstants
+import ftl.run.TestRunner
+import kotlinx.coroutines.experimental.runBlocking
+import picocli.CommandLine.Command
+import picocli.CommandLine.Option
+import java.nio.file.Paths
+
+@Command(
+        name = "cancel",
+        sortOptions = false,
+        headerHeading = "",
+        synopsisHeading = "%n",
+        descriptionHeading = "%n@|bold,underline Description:|@%n%n",
+        parameterListHeading = "%n@|bold,underline Parameters:|@%n",
+        optionListHeading = "%n@|bold,underline Options:|@%n",
+        header = ["Cancels the last iOS Firebase Test Lab run"],
+        description = ["""Selects the most recent run in the results/ folder.
+Reads in the matrix_ids.json file. Cancels any incomplete matrices.
+"""]
+)
+class IosCancelCommand : Runnable {
```

Comment:
The cancel logic should be the same for both Ios and Android, I think we could have one command for both.

Are you sure we need to load in the IosArgs? The folders are named using timestamps so we should be able to pick the latest folder and then read the matrix_ids.json without needing a flank.yml

Reply:
That is an idea. I basically copied the refresh command and its logic flow. I will look into creating one cancel command.